src/azure-cli/azure/cli/command_modules/acr/token.py

```diff
@@ -21,12 +21,15 @@ def acr_token_create(cmd,
                      repository_actions_list=None,
                      status=None,
                      resource_group_name=None,
-                     no_passwords=None):
+                     no_passwords=None,
+                     days=None):
     from knack.log import get_logger
     from ._utils import get_resource_id_by_registry_name
 
     if bool(repository_actions_list) == bool(scope_map_name):
         raise CLIError("usage error: --repository | --scope-map-name")
+    if no_passwords and days is not None:
+        raise CLIError("usage error: --no-passwords and --days are mutually exclusive.")
```

Comment:
What if no_passwords is not None and days? Will CLI throw an error?

Reply:
if `no_password` is False or just missing, then it is a valid case to have `days` for user specified expiry time, or `9999-12-31` otherwise